neural_structured_learning/keras/adversarial_regularization.py

```diff
@@ -20,6 +20,8 @@
 import collections
 import functools
 import types
+from keras.utils import traceback_utils
```

Comment:
Please move these two import lines to line 27 (after "import attr") and swap their order. The idea is that the imports from the standard library goes before the imports from third-party libraries, and the imports in the same group are ordered alphabetically.